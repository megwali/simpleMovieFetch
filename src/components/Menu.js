import { NavLink } from 'react-router-dom';
import { Container } from './StyledComponents';


const Menu = () => (
  <Container>
    <NavLink activeStyle={{ color: 'blue' }} exact to="/">Home</NavLink>
    <NavLink
      activeStyle={{ color: 'blue' }}
      isActive={(match, location) => match || location.pathname === '/show'}
      to="/search"
    >
      Search
    </NavLink>
  </Container>
)

export default Menu;
