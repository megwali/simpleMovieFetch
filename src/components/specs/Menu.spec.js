import { create } from 'react-test-renderer';
import Menu from '../Menu';

jest.mock('react-router-dom', () => ({ NavLink: 'NavLink' }));
jest.mock('../StyledComponents', () => ({ Container: 'Container' }));


describe('Menu', () => {
  it('renders', () => {
    const component = create(<Menu />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
