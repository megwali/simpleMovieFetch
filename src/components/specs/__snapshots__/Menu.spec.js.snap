// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Menu renders 1`] = `
<Container>
  <NavLink
    activeStyle={
      Object {
        "color": "blue",
      }
    }
    exact={true}
    to="/"
  >
    Home
  </NavLink>
  <NavLink
    activeStyle={
      Object {
        "color": "blue",
      }
    }
    isActive={[Function]}
    to="/search"
  >
    Search
  </NavLink>
</Container>
`;
